frontend/src/themes/components/radio.ts

```diff
@@ -9,6 +9,7 @@ const Radio = {
     },
     label: {
       textStyle: "smallerParagraph",
+      pointerEvents: "none",
```

Comment:
what is intended behaviour? 👀

Reply:
produces an error if it is not there - I think the error is within MUI itself (since I saw that many people have the same issue with MUI) but this is one fix that I saw

Reply:
🤔 what is MUI lol

Reply:
OOPS I MEANT CHAKRA UI sorrry lol i keep getting material ui and chakra ui messed up

Reply:
haha all good, sounds good!